docs/UserGuide.adoc

```diff
@@ -154,7 +154,7 @@ a person and the criteria for valid inputs:
 
 === Viewing help : `help`
 There may be times where you do not understand how ORGANice works, or you need a simple reminder on how to use
-ORGANice. This command allows you to access the user guide to ORGANice.
+ORGANice. To access the user guide of ORGANice, do the following:
```

Comment:
maybe can add in "**you can** do the following:"

Reply:
Changed